src/main/java/huyang/Deadline.java

```diff
@@ -0,0 +1,77 @@
+package huyang;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents a deadline task with a specified deadline time.
+ */
+public class Deadline extends Task {
+    protected LocalDateTime by;
```

Comment:
Perhaps the variable name can be deadineDate instead for clarity?

Reply:
Yes, that should make my code clearer! Thank you.